app/org/maproulette/framework/model/Challenge.scala

```diff
@@ -134,6 +134,7 @@ case class ChallengeExtra(
     taskStyles: Option[String] = None,
     taskBundleIdProperty: Option[String] = None,
     isArchived: Boolean = false,
+    reviewSettings: Boolean = false,
```

Comment:
The variable `reviewSettings` can be named better. A "review settings" is confusing since a "settings" (general interpretation) can have many options and naming this `reviewSettings` suggests many "settings" and yet it is a boolean. It's misleading to the reader and I thought `boolean` was a typo.

I think better is `tasksNeedReview` or `defaultTaskNeedReview` or `newTasksRequireReview` (is that the best?) or some other name scheme.